ratpack-cassandra/src/main/java/smartthings/ratpack/cassandra/CassandraModule.java

```diff
@@ -139,8 +161,93 @@ public void setPassword(String password) {
 
 	@Override
 	protected void configure() {
+		bind(Session.class).to(DefaultSession.class).in(Scopes.SINGLETON);
```

Comment:
Rather than subclass this module another possible option is to use the Guice OptionalBinder.  I hadn't used it myself until today, but it seems pretty convenient.  Then you can do it all in your own module rather than creating a new subclass.

```
		OptionalBinder.newOptionalBinder(binder(), AccountService.class)
			.setDefault().to(DefaultAccountService.class).in(Scopes.SINGLETON);
```